cartridge/graphql/execute.lua

```diff
@@ -248,26 +248,22 @@ local function getFieldEntry(objectType, object, fields, context)
     argumentMap[argument.name.value] = argument
   end
 
-  local defaultValues = {}
   if context.operation.variableDefinitions ~= nil then
+    context.variables = context.variables or {}
     for _, value in ipairs(context.operation.variableDefinitions) do
-      if value.defaultValue ~= nil then
-        local variableType = query_util.typeFromAST(value.type, context.schema)
-        defaultValues[value.variable.name.value] = util.coerceValue(value.defaultValue, variableType)
+      local variable_name = value.variable.name.value
+      if type(context.variables[variable_name]) == 'nil' then
+        if value.defaultValue ~= nil then
```

Comment:
Maybe `if type(context.variables[variable_name]) == 'nil' and value.defaultValue ~= nil`?

Reply:
Yes, it's better